src/pybind/mgr/dashboard_v2/controllers/pool.py

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+
+from ..services.ceph_service import CephService
+from ..tools import ApiController, RESTController, AuthRequired
+
+
+@ApiController('pool')
+@AuthRequired()
+class PoolController(RESTController):
```

Comment:
Either `s/PoolController/Pool/` or we should add the `Controller` postfix to all controllers.